packages/neon_framework/lib/widgets.dart

```diff
@@ -7,7 +7,7 @@ export 'package:neon_framework/src/widgets/image.dart' hide NeonImage;
 export 'package:neon_framework/src/widgets/linear_progress_indicator.dart';
 export 'package:neon_framework/src/widgets/list_view.dart';
 export 'package:neon_framework/src/widgets/relative_time.dart';
-export 'package:neon_framework/src/widgets/rich_text/rich_text.dart';
 export 'package:neon_framework/src/widgets/server_icon.dart';
 export 'package:neon_framework/src/widgets/user_avatar.dart' hide NeonUserStatusIndicator;
 export 'package:neon_framework/src/widgets/user_status_icon.dart';
+export 'package:neon_rich_text/neon_rich_text.dart';
```

Comment:
Should we really re-export this package?
What stands against just letting the consumers import it?

Reply:
I mainly did it because we also export parts of account_repository and I felt this makes it easier for consumers, but in the end it is better to not do this to avoid that we accidentally create a cyclic dependency at some point.